### ISSUE TITLE
MaterialImage with type MATERIALBOXES

### ISSUE THREAD

**Issue by nseb** [GwtMaterialDesign/gwt-material#862]
I have several images in my page, the image below to display it I have to scroller, once visible, if I click on it, to have it big, I have this error

VM175:3 Uncaught TypeError: Cannot read property 'css' of undefined
    at Array.complete (<anonymous>:3:1415)
    at d (<anonymous>:1:11738)
    at c (<anonymous>:1:10837)

**Comment by kevzlou7979** [comment]
Nice catch, I will be fixing this an put this on 2.1.1 snapshot

**Comment by nseb** [comment]
Thanks. 
I have another question : on my phone (android) , it's not possible to zoom the image after I click to have it big. (like facebook)

**Comment by kevzlou7979** [comment]
You mean pinching your phone to zoom it? well if that so,  I tried on my phone tapping it to open then pinching it to zoom the image.

**Comment by nseb** [comment]
No, I Tape , the picture becomes big, it's ok. But, once big, I would zoom with two fingers on the phone, in case the image is very large to have details.
